app/facades/paginate_facade.rb

```diff
@@ -0,0 +1,33 @@
+class PaginateFacade
+
+  def get_gutenberg_book(guten_id)
+    JestamouseService.new.get_book_text(guten_id)
+  end
+
+  def remove_extra_returns(raw_text)
+    raw_text.gsub(/(\S)\r\n(\S){1}/, '\1 \2')
+  end
+
+  def paginate_text(clean_text)
+    paginated_book = []
+    until clean_text.empty?
+      text_set = clean_text.truncate(1000, separator: /\s/, omission:'')
+      paginated_book.push(text_set)
+      clean_text.sub!(text_set, '')
+    end
+    return paginated_book
+  end
+
+  def paginated_gutenberg(guten_id)
+    raw_text ||= get_gutenberg_book(guten_id)
+    clean_text ||= remove_extra_returns(raw_text)
```

Comment:
Can you explain what the `||=` is for here. At first I thought you were trying to memoize the `raw_text` and `clean_text` variables, but I don't think one-line memoization works if there is code that gets run after it and I don't see `paginated_gutenberg` getting called more than one time here.

Reply:
Removed memorization. Initially thought until loop in paginated text would keep hitting these variables.